cmake/external/mkldnn.cmake

```diff
@@ -7,31 +7,44 @@ set(MKLDNN_SOURCE ${CMAKE_CURRENT_BINARY_DIR}/mkl-dnn/src/mkl-dnn/src)
 set(MKLDNN_INSTALL ${CMAKE_CURRENT_BINARY_DIR}/mkl-dnn/install)
 set(MKLDNN_LIB_DIR ${MKLDNN_INSTALL}/lib)
 set(MKLDNN_INCLUDE_DIR ${MKLDNN_INSTALL}/include)
+# Note that we can't find the header from cmake, because mklml external
```

Comment:
maybe extend the note in line 4, that the MKL_VERSION may need to be updated as well if MKLDNN_TAG is updated.